src/viam/utils/visualization/graph.go

```diff
@@ -0,0 +1,393 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/spatialmath"
+
+	"github.com/rocketlaunchr/dataframe-go/imports"
+	chart "github.com/wcharczuk/go-chart"
```

Comment:
Why are we using a second package to plot?

Reply:
I think this is the desired way to do plotting, linked [here](https://github.com/rocketlaunchr/dataframe-go/?tab=readme-ov-file#plotting-cross-platform) is an example from the README of `dataframe-go`